### DISCUSSION TITLE
[Feature] Add captcha to login page

### DISCUSSION THREAD

**Discussion by Rajdave69** [immich-app/immich#11118] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Add a captcha to the login page, this will stop automated bruteforce attacks.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
The Immich login system is deliberately very minimal. If you want more advanced features, use oauth, or for a case like this a tool like fail2ban could also work.
